client/src/components/ui/forms/document/CreateDocument.svelte

```diff
@@ -8,15 +8,18 @@
     import { dashboardState } from '../../../../pages/dashboard/stores/DashboardState.ts';
     import { reloadMetrics } from '../../../../pages/dashboard/stores/MetricStore.ts';
     import { topToastMessage } from '../../../../pages/dashboard/stores/ToastStore.ts';
+    import { deferredSnaps } from '../../../../pages/dashboard/stores/DeferredStore.ts';
 
     import type { Category } from '../../../../.../../../../model/src/category.ts';
     import type { Document } from '../../../../.../../../../model/src/document.ts';
-    import type { Snapshot } from '../../../../.../../../../model/src/snapshot.ts';
+    import { Snapshot, Status } from '../../../../.../../../../model/src/snapshot.ts';
 
     import Button from '../../Button.svelte';
     import BarcodeSelect from '../../BarcodeSelect.svelte';
     import CategorySelect from '../../CategorySelect.svelte';
     import TextInput from '../../TextInput.svelte';
+    import { upsert } from './utils.ts';
+    import { DeferredSnap } from '../../../../api/error.ts';

```

Comment:
Let's space out the imports.
```suggestion

    import { upsert } from './utils.ts';
    import { DeferredSnap } from '../../../../api/error.ts';
```